content/docs/sidebar.json

```
@@ -388,7 +388,8 @@
           {
             "label": "plots modify",
             "slug": "modify"
-          }
+          },
+          { "label": "plots templates", "slug": "templates" }
```

minor: style looks different?

---

Yeah I'm surprised the linters didn't complain. I guess we could use compressed lines like this but... for now being consistent (committing):
```suggestion
          {
            "label": "plots templates",
            "slug": "templates"
          }
```